api.js

```diff
@@ -224,3 +236,86 @@ Api.prototype._run = function (files, _options) {
 		return runStatus;
 	});
 };
+
+Api._blankResults = {
```

Comment:
I think this should be a getter that returns the object. It might be an edge case, but if the consumer modifies the object, later results will also be modified. I might be missing something though and this might be a non-issue, but thought I would bring it up.